cli/cli_set_analytics.go

```diff
@@ -3,6 +3,7 @@ package cli
 import (
 	"context"
 	"fmt"
+	"os"
```

Comment:
Ignore the changes in this file. The actual consent flow is implemented [in that PR](https://github.com/NordSecurity/nordvpn-linux/pull/904) and this PR should be merged after it